src/ParameterDefinition.ts

```diff
@@ -2,6 +2,8 @@
 // Copyright (c) Microsoft Corporation.  All rights reserved.
 // ----------------------------------------------------------------------------
 
+// tslint:disable:no-unused-expression
```

Comment:
nit: What's the reason for this? I like other cases where you put "// Grandfathered in" (assuming that's the reason)